test/kubemark/resources/manifests/addons/kubemark-rbac-bindings/service-account-binding.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: rbac.authorization.k8s.io/v1alpha1
+kind: ClusterRoleBinding
+metadata:
+  name: service-account-cluster-admin
+  labels:
+    kubernetes.io/cluster-service: "true"
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: cluster-admin
```

Comment:
Similarly here I thought this binding was being used by the kube-controller-manager. But infact, removing this also doesn't break anything.
Why is this binding not being used or is there some other service account that the controller-manager is using by default? ([Ref](https://github.com/shyamjvs/kubernetes/blob/e1c6ad9fc4a095984789d9ed1f8ff79cb03619d3/test/kubemark/resources/start-kubemark-master.sh#L410))